cli/bench/lsp.rs

```diff
@@ -0,0 +1,274 @@
+// Copyright 2018-2021 the Deno authors. All rights reserved. MIT license.
+
+use deno_core::error::generic_error;
+use deno_core::error::AnyError;
+use deno_core::serde::de;
+use deno_core::serde::Deserialize;
+use deno_core::serde_json;
+use deno_core::serde_json::json;
+use deno_core::serde_json::Value;
+use lazy_static::lazy_static;
+use regex::Regex;
+use std::collections::HashMap;
+use std::path::PathBuf;
+use std::process::Stdio;
+use tokio::io::AsyncBufReadExt;
+use tokio::io::AsyncReadExt;
+use tokio::io::AsyncWriteExt;
+use tokio::io::BufReader;
+use tokio::io::BufWriter;
+use tokio::io::Lines;
+use tokio::process::ChildStderr;
+use tokio::process::ChildStdin;
+use tokio::process::ChildStdout;
+use tokio::process::Command;
+use tokio::runtime;
+use tokio::time::Instant;
+
+lazy_static! {
+  static ref CONTENT_TYPE_REG: Regex =
+    Regex::new(r"(?i)^content-length:\s+(\d+)").unwrap();
+}
+
+#[derive(Debug, Deserialize)]
+struct LspResponse {
+  pub result: Option<Value>,
+  pub id: u32,
+  pub error: Option<LspResponseError>,
+}
+
+#[derive(Debug, Deserialize)]
+struct LspResponseError {
+  pub code: i32,
+  pub message: String,
+  pub data: Option<Value>,
+}
+
+#[derive(Debug, Deserialize)]
+struct LspNotification {
+  pub method: String,
+  pub params: Option<Value>,
+}
+
+fn create_basic_runtime() -> runtime::Runtime {
+  runtime::Builder::new_current_thread()
+    .enable_io()
+    .enable_time()
+    // for performance purposes, we might as well allow this to go to as
+    // many threads as it can, so that the harness doesn't become the bottle
+    // neck
+    // .max_blocking_threads(32)
+    .build()
+    .unwrap()
+}
```

Comment:
Why not use `tokio_util::create_basic_runtime()`?

Reply:
🤦  I thought it was in a different crate, but I guess it isn't... silly me.